hermeto/core/package_managers/javascript/yarn/project.py

```
@@ -125,7 +139,57 @@ def from_source_dir(cls, source_dir: RootedPath) -> "Project":
+        project.yarn_rc.get("pnpDataPath"): "pnpDataPath",
+        project.yarn_rc.get("pnpUnpluggedFolder"): "pnpUnpluggedFolder",
+        project.yarn_rc.get("installStatePath"): "installStatePath",
+        project.yarn_rc.get("patchFolder"): "patchFolder",
+        project.yarn_rc.get("virtualFolder"): "virtualFolder",
+    }
+
+    for path in paths_conf_opts:
+        if path is not None:
+            try:
+                project.source_dir.join_within_root(path)
+            except Exception:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)